examples/tk-gas-station/src/transferUSDCDelegated.ts

```diff
@@ -0,0 +1,198 @@
+  BASE_RPC_URL: z.string().url(),
+});
+
+const env = envSchema.parse(process.env);
+
+print(
+  `🌐 Using ${selectedChain.toUpperCase()} network`,
+  `USDC: ${preset.tokens?.USDC}`,
+);
+
+const turnkeyClient = new TurnkeyServerSDK({
+  apiBaseUrl: env.BASE_URL,
+  apiPrivateKey: env.API_PRIVATE_KEY,
+  apiPublicKey: env.API_PUBLIC_KEY,
+  defaultOrganizationId: env.ORGANIZATION_ID,
+});
+
+/**
+ * Demonstrates USDC transfer using the Gas Station pattern with EIP-7702 authorization
+ *
+ * This example shows the separation of concerns:
+ * 1. End user client signs the authorization and creates signed intents
+ * 2. Paymaster client submits transactions and pays for gas
+ */
+const main = async () => {
+  const rpcUrl = selectedChain === "base" ? env.BASE_RPC_URL : env.ETH_RPC_URL;
+
+  // Create viem wallet clients with Turnkey accounts
+  const userAccount = await createAccount({
+    client: turnkeyClient.apiClient(),
+    organizationId: env.ORGANIZATION_ID,
+    signWith: env.EOA_ADDRESS as `0x${string}`,
+  });
+
+  const paymasterAccount = await createAccount({
+    client: turnkeyClient.apiClient(),
+    organizationId: env.ORGANIZATION_ID,
+    signWith: env.PAYMASTER_ADDRESS as `0x${string}`,
+  });
+
+  const userWalletClient = createWalletClient({
+    account: userAccount,
+    chain: preset.chain,
+    transport: http(rpcUrl),
+  });
+
+  const paymasterWalletClient = createWalletClient({
+    account: paymasterAccount,
+    chain: preset.chain,
+    transport: http(rpcUrl),
+  });
+
+  // Create Gas Station clients with the viem wallet clients
+  const userClient = new GasStationClient({
+    walletClient: userWalletClient,
+  });
+
+  const paymasterClient = new GasStationClient({
+    walletClient: paymasterWalletClient,
+  });
+
+  // Explorer URL for displaying transaction links
+  const explorerUrl =
+    selectedChain === "base" ? "https://basescan.org" : "https://etherscan.io";
+
+  // Step 1: Check if EOA is already delegated, authorize if needed
+  const isDelegated = await userClient.isDelegated();
+
+  if (!isDelegated) {
+    print("===== Starting EIP-7702 Authorization =====", "");
+    print("EOA not yet delegated", "Starting authorization...");
+    print("User signing authorization...", "");
+    const authResult = await userClient.authorize(paymasterClient);
+    print("Authorization transaction sent", authResult.txHash);
+    print("Waiting for confirmation...", "");
+    print("✅ Authorization SUCCEEDED", "");
+    print("Verifying delegation on-chain...", "");
+    print("✓ Delegation verified on-chain", "");
+    print(
+      "✅ Authorization complete",
+      `${explorerUrl}/tx/${authResult.txHash}`,
+    );
+  } else {
+    print("✓ EOA already delegated", "Skipping authorization");
+  }
+
+  // Step 2: Execute USDC transfer using the generic execute API with helpers
+  print("===== Starting USDC Transfer =====", "");
+
+  const transferAmount = parseUnits("0.01", 6); // 1 penny in USDC (6 decimals)
```

Comment:
tiny nit: we can define the amount (0.01) as a const somewhere (or configurable var) and prevent having to repeat here and in the print statement below